src/lib/resources/cronjob/timeout.ts

```diff
@@ -0,0 +1,7 @@
+export function checkTimeout(timeout: number) {
+  if (timeout > 36800) {
```

Comment:
36800 seconds is 10.2 hours, not 24. This should be 86400.

Or going even further, I'd recommend sticking with the specialized data type as long as possible:

```suggestion
const maxDuration = Duration.fromString("24h");

export function checkTimeout(timeout: Duration) {
  if (duration.compare(maxDuration) > 0) {
```